entity-framework/core/performance/advanced-performance-topics.md

```diff
@@ -77,6 +77,12 @@ The full source code for this sample is available [here](https://github.com/dotn
 > [!NOTE]
 > Although EF Core takes care of resetting internal state for `DbContext` and its related services, it generally does not reset state in the underlying database driver, which is outside of EF. For example, if you manually open and use a `DbConnection` or otherwise manipulate ADO.NET state, it's up to you to restore that state before returning the context instance to the pool, e.g. by closing the connection. Failure to do so may cause state to get leaked across unrelated requests.
 
+### Connection Pooling Considerations
+
+Connection pooling in Entity Framework is managed at the client level through ADO.NET and works consistently across all SQL Server-based environments, including Azure SQL Database, Azure SQL Managed Instance, and SQL Server (on-premises or in virtual machines). While pooling mechanisms are consistent, service-specific factors may impact pooling efficiency. Entity Framework relies on ADO.NET for database interactions, including connection pooling, which reuses existing database connections to minimize the overhead of repeatedly opening and closing connections.
```

Comment:
I agree that it makes sense to discuss connection pooling and its relationship (or rather non-relationship) to EF. Though I'd avoid saying "connection pooling in EF" (since there is no connection pooling at EF, as you do explain). I think it's also inappropriate to focus on SQL Server here specifically, as the information is generally valid to all databases.

I'll take a look a bit later and probably tweak the wording a bit, but otherwise the gist is good - thanks for submitting this!